venues/ICLR.cc/2020/Conference/python/reassign-reviewer.py

```diff
@@ -0,0 +1,108 @@
+## Import statements
+import argparse
+import sys
+import csv
+import openreview
+import re
+
+'''
+Requirements:
+
+openreview-py
+
+Usage:
+
+Use the --paper (-p) flag to specify the paper number.
+Use the --add (-a) flag to specify a username or email address to assign.
+Use the --remove (-r) flag to specify a username or email address to remove.
+
+The script processes removals before additions, and assigns the user to the
+lowest AnonReviewer# group that is empty.
+
+For example, after running the following:
+
+python assign-reviewer.py --paper 123 --remove ~Oriol_Vinyals1 --add ~MarcAurelio_Ranzato1
+
+
+Paper123/Reviewers = {
+    AnonReviewer1: ~Tara_Sainath1
+    AnonReviewer2: ~Oriol_Vinyals1
+    AnonReviewer3: ~Iain_Murray1
+}
+
+becomes
+
+Paper123/Reviewers = {
+    AnonReviewer1: ~Tara_Sainath1
+    AnonReviewer2: ~MarcAurelio_Ranzato1
+    AnonReviewer3: ~Iain_Murray1
+}
+'''
+
+if __name__ == "__main__":
+    ## Argument handling
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-p','--paper', required=True)
+    parser.add_argument('-a','--add')
+    parser.add_argument('-r','--remove')
+    parser.add_argument('--baseurl', help="base url")
+    parser.add_argument('--username')
+    parser.add_argument('--password')
+
+    args = parser.parse_args()
+
+    client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+    paper_number = args.paper
+    reviewer_to_remove = args.remove
+    reviewer_to_add = args.add
+
+    conference = 'ICLR.cc/2020/Conference'
+    paperUrl = "ICLR.cc/2020/Conference/Paper{}".format(paper_number)
+    unsubmittedGroupId = paperUrl + "/Reviewers/Unsubmitted"
```

Comment:
We don't have un-submitted groups anymore.

Reply:
No, only submitted groups. I can remove this variable